minvws/nl-kat-coordination
# Refactor KATalogus client in Rocky

### Changes

- Handles client errors generically in a base client that can be organization-agnostic for some methods. 
- Introduces a new intermediate "adapter" called KATalogus that forces you to set an organization.

### Issue link

Closes https://github.com/minvws/nl-kat-coordination/issues/3697

### Demo

-

### QA notes

Please verify that we can still properly create organizations, add plugins, settings and enable/disable them from Rocky!

---

### Code Checklist


---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
